docs/configure.rst

```diff
@@ -192,6 +192,9 @@ For example, if we have set it to ``warning``, no debug or informational message
 Finally, there is a special set of handlers for handling performance log messages.
 These are stored in the ``handlers_perflog`` property.
 The performance handler in this example will create a file per test and per system/partition combination and will append the performance data to it every time the test is run.
+The path to the file is created as follows: ``<basedir>/<prefix>/<testname>log``.
+``<basedir>`` is controlled by the ``--perflogdir`` option, the environment variable :envvar:`RFM_PERFLOG_DIR` or the ``basedir`` configuration parameter in the ``handlers_perflog``, which is ``.`` by default; ``<prefix>`` is what is defined in the ``handlers_perflog``.
```

Comment:
```suggestion
``<basedir>`` is controlled by the ``--perflogdir`` option, the environment variable :envvar:`RFM_PERFLOG_DIR` and the ``basedir`` configuration parameter in the ``handlers_perflog``, which is ``.`` by default; ``<prefix>`` is defined in the ``handlers_perflog``.
```

Reply:
Imho, all this description is not needed. It described in the configuration reference of the `filelog` handler. So this is a duplication. We do have a link to the reference at the end of this section. The key problem in the documentation, in my opinion, is that we don't say explicitly that performance logging happens *only* for performance tests.